auditlogger.go

```diff
@@ -0,0 +1,15 @@
+package ladon
+
+// AuditLogger tracks denied and granted authorizations.
+type AuditLogger interface {
+	OnDeniedAuthorization(request *Request, pool Policies, deciders Policies)
+	OnGrantedAuthorization(request *Request, pool Policies, deciders Policies)
```

Comment:
What do you think about `LogGrantedAuthorization` or `LogAuthorizationGranted` or `LogGrantedAccessRequest`?